css/index.css

```
@@ -1,3 +1,286 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+/* Code starts here */
+.container {
+	width: 100%;
+	margin: auto 0;
+}
+
+header {
+    width: 100%;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    display: flex;
+    flex-direction: row;
+    float: left; 
+    justify-content: center;
+}
+
+nav a {
+    color: lightgray;
+    text-align: center;
+    justify-content: center;
+    text-decoration: none;
+    margin-top: 40px;
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-bottom: 50px;
+  }
+
+  nav a .logo {
+    margin-left: 40px
+  }
+  
+  .top-container {
+    width: 90%;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-direction: row;
+    margin-bottom: 30px;
+	margin-left: 30px;
+    margin-top: 15px;
+  }
+
+  .top-container .item {
+    width: 40%;
+    margin: 20px;
+    align-items: center;
+    justify-items: center;
+    
+  }
+  .button-div {
+    /* display: flex; */
+    align-items: center;
+    justify-content: center;
+  }
+  .top-container button {
+    
+    padding: 5px 40px 5px;
+	border: 1px solid black;
+    font-family: 'Titillium Web';
+    margin-bottom: 15px;
+    margin-top: 15px;	
+    position:relative;
+    left:40%;
+  }
+
+  h1 {
+    font-size: 80px;
+    text-align: center;
+    margin: 0 auto;
+  }
+
+  .top-p-container {
+    width: 90%;
+	display: flex;
+	justify-content: space-evenly;
+    align-items: center;
+    flex-direction: row;
+	flex-wrap: wrap;
+	margin-bottom: 30px;
+	margin-left: 30px;
+    margin-top: 15px;
+    padding-top: 30px;
```

```suggestion
padding: 30px 0 20px;
```
You could combine both of these padding properties into one line